source/install/desktop-managed-resources.rst

```diff
@@ -0,0 +1,40 @@
+
+Desktop - Managed Resources
+===========================
+
+The Mattermost Desktop App supports managed resources. A managed resource can be any service available on the same hostname using the same protocol as the Mattermost server.
+
+.. Note::
+    Using this feature requires a `custom build of the Mattermost Desktop App <https://docs.mattermost.com/deployment/desktop-app-deployment.html>`_.
+
+When adding the path of a managed resource to `src/common/config/buildConfig.js <https://github.com/mattermost/desktop/blob/master/src/common/config/buildConfig.js>`_ it opens as a pop-up window in the Mattermost Desktop App. 
```

Comment:
Would the user add the path to something in this repo? Or would it be in their own config file? If the latter, one suggestion is to replace the URL with ``buildConfig`` in the ``config.json`` file (replace with correct config setting  of course) and maybe a link to the config doc?

Reply:
that would be on their own clone of the repo, it shouldn't appear in a PR
the reason for it being in buildConfig is to prevent users from modifying as this is for admins to provide safe resources

Reply:
Thanks, @Willyfrog. That makes sense! So they'd need to be a Mattermost System Admin in order to do this? Or general admin for their environment?

Reply:
A general admin in their org, someone who knows how's the network of their org setup to make the appropriate changes

Reply:
In that case, one option is to include an example of what they'd change and where. 

One other question is around the pop-up window. This seems like an important aspect of the content. Would it be an option to expand on it in the opening paragraph with an outline of the benefits or explanation of why one would want that?

Reply:
```suggestion
Add the path of a managed resource to your configuration file. When selected, it opens as a pop-up window in the Mattermost Desktop App. 
```